src/GUI/RevEng.Shared/Cli/CliConfigMapper.cs

```diff
@@ -158,6 +154,37 @@ public static bool TryGetCliConfig(string fullPath, string connectionString, Dat
             return true;
         }
 
+        /// <summary>
+        /// Ensures that any excluded columns for tables are not required. Removes the invalid columns from the list.
+        /// </summary>
+        /// <param name="config">Configuration to Validate.</param>
+        /// <param name="objects">Table Models to check against.</param>
+        private static List<string> ValidateExcludedColumns(CliConfig config, List<TableModel> objects)
+        {
+            List<string> warnings = new();
+
+            var objectsToCheck = config.Tables.Where(x => x.ExcludedColumns?.Count > 0)
+                .Select(table => table as IEntity)
+                .Union(config.Views.Where(x => x.ExcludedColumns?.Count > 0));
+
+            foreach (var table in objectsToCheck)
+            {
+                var dbTable = objects.Single(x => x.DisplayName == table.Name);
+                var columnsThatCannotBeExcluded = dbTable.Columns.Where(x => x.IsForeignKey || x.IsPrimaryKey).Select(x => x.Name.ToUpperInvariant());
```

Comment:
I think some database systems support case senstitive schemas, so this should propably remain case sensitive